### PR TITLE
Add development language names to the landing page

### LINKED ISSUE #276: Landing page: mention languages available to write KW policies
On the landing page we should make explicit which programming and scripting languages can be used to write KW policies. Right now this is all about WASM, which is great, but not so clear to our end users.

### PR DESCRIPTION
**Pull request by jhkrug**
## Description

Will close #276.

### CODE REVIEW

**Comment by jhkrug** [comment]
Language names are links. Necessary? Correct? Better pointing into our docs?

**Review by jvanz — APPROVED** [review]
LGTM

File: content/_index.html [hunk 1]
```diff
@@ -35,6 +35,7 @@ <h4>As a Policy author</h4>
       <p class="text-left font-weight-light">Kubewarden doesn't force you to learn a new Domain Specific Language
       or a query language.</p>
       <p class="text-left font-weight-light">Write policies in your favorite programming language. Reuse your skills and feel instantly productive.
+      <p class="text-left font-weight-light">Develop policies in <a href="https://rust-lang.org" target="_blank">Rust</a>, <a href="https://go.dev" target="_blank">Go</a>, <a href="https://cel.dev" target="_blank">CEL</a>, <a href="https://www.openpolicyagent.org/docs/latest/policy-language/" target="_blank">Rego</a>, and even <a href="https://docs.kubewarden.io/tutorials/writing-policies/wasi/raw-policies" target="_blank">raw policies</a>, described as JSON.
```
Comment (flavio) [inline content/_index.html:38]: I would drop the line about the raw policies. They are still written using Go, Rust, CEL, Rego,... but they target JSON documents, they are not written using JSON.

This information (the raw policies) should be shown somewhere else, the key point would be: "Kubewarden can also be used to validate non-Kubernetes admission requests. Its "raw" policies can be used to validate any kind of JSON document"

**Review by flavio — CHANGES_REQUESTED** [review]
Thanks for the PR, I left a technical note

File: content/_index.html (hunk 1 above)
Comment (jhkrug) [reply content/_index.html]: Thanks @flavio. Made some chnages. What does everybody think now? 

- Are the links OK, or should they point to our internal tutorial docs?
- I added the raw policy stuff to an 'Adaptable' section in 'Why use Kubewarden'. Options for 'Adaptable' might be 'General purpose', 'Wide applicability', etc. 'Adaptable' seemed pithy. Thoughts?

**Review by flavio — APPROVED** [review]
Thanks, this looks great!

**Pull request merged**